Machine Learning module improvements and fixes

cbethell opened the pull request:
This commit addresses [PR#133](https://github.com/AlexsLemonade/training-modules/issues/133) and part of [PR#112](https://github.com/AlexsLemonade/training-modules/issues/112) as follows:

- More context was added to the comment above the mean comparison of the duplicated symbol in the 01-data-prep nb.
- Comments were added to each code chunk related to getting the class labels out of the cluster results in 02-clustering nb (for more context).
- The first instances of the abbreviations `FDR` and `AUC` were spelled out for further context in the 03-PLIER nb.
-  "The strong associations between ribosome and LV90" was changed to "...LV5" to be more relevant to the data in the PLIER nb.
- The step to load `ComplexHeatmap` was taken out of the PLIER nb as it is not used. 
- The PCL files were changed to TSV and changed in the nb's where they are used.
- Re-ran notebooks

Left unaddressed from both PR's:
- My suggestion to explicitly name the pathway in the table that corresponds to the LV index number we want does not seem necessary after taking another look at the 05-exercise nb. It is clear that we want the LV index that corresponds with the proteasome pathway, which should make it simple enough for participants to locate. 
- In [PR#112](https://github.com/AlexsLemonade/training-modules/issues/112#issue-460887024), it is not clear where `all.equal` should be included. 
- In [PR#112](https://github.com/AlexsLemonade/training-modules/issues/112#issuecomment-505841653), I agree with @cansav09 that the exercise instructions involving the function `summary` versus PLIER object's `$summary` are pretty clear. Any further thoughts on how to make them clearer?

cansav09 commented:
> * My suggestion to explicitly name the pathway in the table that corresponds to the LV index number we want does not seem necessary after taking another look at the 05-exercise nb. It is clear that we want the LV index that corresponds with the proteasome pathway, which should make it simple enough for participants to locate.

I'm not 100% sure what the suggestion is here, but we want the participants to be able to navigate the PLIER results as they come out of the package, so if they come with those numbers and names, we kinda need to stick with those. I also may completely be misunderstanding what you are suggesting. 

> * In [PR#112](https://github.com/AlexsLemonade/training-modules/issues/112#issue-460887024), it is not clear where `all.equal` should be included.

~Line 153 in 02-medulloblastoma-clustering, there is an `==` that should be converted to `all.equal` method instead. 
Also, when you are confused about what anyone means on an issue, you can mention things like this on the issue so we can get it figure out before the PR. 👍 

> * In [PR#112](https://github.com/AlexsLemonade/training-modules/issues/112#issuecomment-505841653), I agree with @cansav09 that the exercise instructions involving the function `summary` versus PLIER object's `$summary` are pretty clear. Any further thoughts on how to make them clearer?

That might just be a comment we keep in mind for presenting but we don't necessarily make explicit curriculum changes for. It was only one person I think that had that confusion, so it's not a huge deal.

cansav09 commented:
> This commit addresses PR#133 and part of PR#112 as follows:

Also I think you mean this *PR* addresses these *issues*?

cbethell commented:
@cansav09 
> I'm not 100% sure what the suggestion is here, but we want the participants to be able to navigate the PLIER results as they come out of the package, so if they come with those numbers and names, we kinda need to stick with those. I also may completely be misunderstanding what you are suggesting.

I was referring to the last suggestion I made in issue #133 ..."In the 05-exercise nb, it may be helpful to clarify in line 96 that the LV index column that participants should be looking for should correspond with the KEGG_PROTEASOME pathway." Very good point though. 

>~Line 153 in 02-medulloblastoma-clustering, there is an == that should be converted to all.equal method instead.
Also, when you are confused about what anyone means on an issue, you can mention things like this on the issue so we can get it figure out before the PR. 👍

Thanks for the clarification and the tip! I'll be sure to do that moving forward. 

> That might just be a comment we keep in mind for presenting but we don't necessarily make explicit curriculum changes for. It was only one person I think that had that confusion, so it's not a huge deal.

This sounds good to me. 

> Also I think you mean this PR addresses these issues?

Yes, you're right. Please excuse my disarranged statement!

cansav09 commented on machine-learning/01-medulloblastoma_data_prep.Rmd:
```diff
@@ -1,9 +1,12 @@
+  html_notebook:
+    toc: yes
+    toc_float: yes
+  html_document:
```
Can we get rid of the added heading here?

cansav09 commented on machine-learning/01-medulloblastoma_data_prep.Rmd:
```diff
@@ -48,12 +51,12 @@ library(org.Hs.eg.db)
 # the file used for mapping between gene ids that is output
 mapping_file <- file.path(data_dir, 
                           "genesymbol_ensembl_mappings_used_for_GSE37382.tsv")
 # the prepped expression data output fule
```
```suggestion
# the prepped expression data output file
```
Who knows how long this typo has been here? ¯\_(ツ)_/¯

cansav09 commented on machine-learning/01-medulloblastoma_data_prep.Rmd:
```diff
@@ -243,7 +246,8 @@ agg_exprs_df %>%
 
-Is this consistent with what we expect?
+Is this consistent with what we expect?  
+Note that the values in each set of parentheses below represent the gene expression value of each CALM3 duplicate for that sample.
```
```suggestion
Note that the values in each set of parentheses below represent the gene expression value of each *CALM3* duplicate for that sample.
```
I could be wrong, but I think generally when we refer to the expression of a gene it needs to italicized? I always get it mixed up though, so double check me on this.

cbethell replied:
You're correct!

cbethell commented:
> Have you re-ran the notebooks to refresh the html output after making the latest changes? If so, then everything LGTM to me!

Yes, I re-ran the notebooks. :+1:

jaclyn-taroni commented on machine-learning/03-medulloblastoma_PLIER.Rmd:
```diff
@@ -210,22 +208,22 @@ plier_summary_df
 
 We can filter this `data.frame` based on significance of associations.
-We'll use `FDR < 0.05`.
+We'll use `FDR < 0.05`, where `FDR` is the False Discovery Rate. 
```
Can we link to:

* [Wikipedia](https://en.wikipedia.org/wiki/False_discovery_rate)
* This [awesome StatQuest video](https://www.youtube.com/watch?v=K8LQSvtjcEo)

Here

jaclyn-taroni commented on machine-learning/03-medulloblastoma_PLIER.Rmd:
````diff
@@ -210,22 +208,22 @@ plier_summary_df
 ```
 
-We can also sort by AUC values after filtering.
+We can also sort by AUC (Area Under the ROC Curve) values after filtering.
````
This might be a good link to go here: https://towardsdatascience.com/understanding-auc-roc-curve-68b2303cc9c5
